### PR TITLE
[13.0][MIG] purchase_invoice_plan

### PR DESCRIPTION
**Pull request by kittiu**
Standard migration, ready for review.

### CODE REVIEW

**Comment by kittiu** [comment]
@ps-tubtim @AaronHForgeFlow would you help review this?

File: purchase_invoice_plan/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+- Create new purchase quotation as per normal process
+- Select option "Use Invoice Plan", a new Invoice Plan tab will appear
+- Click on "=> Create Invoice Plan" link to open invoice planning wizard
+- Do plan for number of installment, start date and interval
+- Double check that each installment has correct plan percentage
+- After confirm purchases order, we have new option to "Create Bill by Plan"
```
Comment (Saran440) [inline purchase_invoice_plan/readme/USAGE.rst:6]:
```suggestion
- Confirm Order > Receive Products, see new option to "Create Bill by Plan"
```

**Review by Saran440 — APPROVED** [review]
Functional Test :+1: 

it will look good with test script :)

File: purchase_invoice_plan/readme/USAGE.rst (hunk 1 above)
Comment (Saran440) [reply purchase_invoice_plan/readme/USAGE.rst:6]: this is my mistake, I config product purchase policy is `On received quantities`
it should be `On ordered quantities`

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Functional tests LGTM

File: purchase_invoice_plan/models/purchase.py [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+from odoo.tools.float_utils import float_compare, float_round
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    invoice_plan_ids = fields.One2many(
+        comodel_name="purchase.invoice.plan",
+        inverse_name="purchase_id",
+        string="Inovice Plan",
```
Comment (HviorForgeFlow) [inline purchase_invoice_plan/models/purchase.py]:
```suggestion
        string="Invoice Plan",
```

**Review by HviorForgeFlow — APPROVED** [review]
Just a misspelling comment

File: purchase_invoice_plan/models/purchase.py (hunk 2 above)
Comment (HviorForgeFlow) [reply purchase_invoice_plan/models/purchase.py]: @kittiu waiting this comment to merge

Comment (kittiu) [reply purchase_invoice_plan/models/purchase.py]: Done.

**Pull request merged**